src/components/hv-list/index.tsx

```diff
@@ -69,7 +66,7 @@ export default class HvList extends PureComponent<HvComponentProps, State> {
   };
 
   getItems = () => {
-    const isOwnedBySelf = item => {
+    const isOwnedBySelf = (item: undefined) => {
```

Comment:
This doesn't seem right - I'm not sure why the `item.parentNode` property access below wouldn't cause a type checking error